tools/pyi/gen_pyi.py

```diff
@@ -576,6 +573,10 @@ def gen_pyi(declarations_path, out):
                ],
         'item': ["def item(self) -> Number: ..."],
         'copy_': ["def copy_(self, src: Tensor, non_blocking: _bool=False) -> Tensor: ..."],
+        'set_': ['def set_(self, storage: Storage, offset: _int, size: Size, stride: Tuple[_int, ...]) -> Tensor: ...',
```

Comment:
Reading this again, I don't think `Tuple` is the right type here, because I'm pretty sure we accept lists too here. I think using the `_size` type for `size` and `stride` will be more correct.

Reply:
`_size` is defined in `torch.types`